### ISSUE TITLE
file delete not completing

### ISSUE THREAD

**Issue by gjost** [densho/ddr-local#212]
pkikawa 2017-08-28 11:17

Error ID:[EID_20170828_1]
Status: [ToDo]
Local Machine: [see comment]
Full Collection+Object ID Error Path: []
Time and Date of Error: [recurring error since at least Aug 1st, most reciently aug 28 at 11:00]
Brief Error Description: [deleting binary files from the webui results in a failure notification, but the files get deleted in the actual repo. runnig git status reveals files have been deleted, but changes to the object entity.json are not staged for commit.]
Steps to Replicate Error:[delete any master/mezz from the web ui (tested and repeatable in testing collections), error messages same]

**Comment by gjost** [comment]
```
Traceback (most recent call last):
  File "/opt/ddr-local/venv/ddrlocal/local/lib/python2.7/site-packages/celery/app/trace.py", line 240, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/opt/ddr-local/venv/ddrlocal/local/lib/python2.7/site-packages/celery/app/trace.py", line 438, in __protected_call__
    return self.run(*args, **kwargs)
  File "/opt/ddr-local/ddrlocal/webui/tasks.py", line 808, in delete_file
    git_name, git_mail, agent
  File "/opt/ddr-local/venv/ddrlocal/local/lib/python2.7/site-packages/ddr_cmdln-0.9.4b0-py2.7.egg/DDR/models/__init__.py", line 2283, in delete
    commit=commit
  File "/opt/ddr-local/venv/ddrlocal/local/lib/python2.7/site-packages/ddr_cmdln-0.9.4b0-py2.7.egg/DDR/commands.py", line 51, in wrapper
    return f(*args, **kwargs)
  File "/opt/ddr-local/venv/ddrlocal/local/lib/python2.7/site-packages/ddr_cmdln-0.9.4b0-py2.7.egg/DDR/commands.py", line 41, in wrapper
    return f(*args, **kwargs)
  File "/opt/ddr-local/venv/ddrlocal/local/lib/python2.7/site-packages/ddr_cmdln-0.9.4b0-py2.7.egg/DDR/commands.py", line 626, in file_destroy
    repo.git.rm('-rf', f)
  File "/opt/ddr-local/venv/ddrlocal/local/lib/python2.7/site-packages/git/cmd.py", line 227, in <lambda>
    return lambda *args, **kwargs: self._call_process(name, *args, **kwargs)
  File "/opt/ddr-local/venv/ddrlocal/local/lib/python2.7/site-packages/git/cmd.py", line 456, in _call_process
    return self.execute(call, **_kwargs)
  File "/opt/ddr-local/venv/ddrlocal/local/lib/python2.7/site-packages/git/cmd.py", line 377, in execute
    raise GitCommandError(command, status, stderr_value)
GitCommandError: 'git rm -rf files/ddr-testing-40032-1/files/ddr-testing-40032-1-master-a23c34b2db.jpg' returned exit status 128: fatal: pathspec 'files/ddr-testing-40032-1/files/ddr-testing-40032-1-master-a23c34b2db.jpg' did not match any files
```

**Comment by gjost** [comment]
Fixed in affaf5c.
